matrix-sdk/src/main/java/org/matrix/androidsdk/HomeServerConnectionConfig.java

```diff
@@ -46,6 +50,12 @@
     private Credentials mCredentials;
     // tell whether we should reject X509 certs that were issued by trusts CAs and only trustcerts with matching fingerprints.
     private boolean mPin;
+    // the accepted TLS versions
+    private List<TlsVersion> mTlsVersions;
+    // the accepted TLS cipher suites
+    private List<CipherSuite> mTlsCipherSuites;
+    // should accept TLS extensions
+    private boolean mTlsExtensions;
```

Comment:
Rename this member to `mShouldAcceptTlsExtensions`

Reply:
Ok